core/src/components/searchbar/searchbar.ios.scss

```diff
@@ -45,7 +45,7 @@
 
 .searchbar-input {
   @include padding(0, 28px);
-  @include border-radius($searchbar-ios-input-border-radius);
+  @include border-radius(var(--border-radius, $searchbar-ios-input-border-radius));
```

Comment:
Instead of doing this as a fallback, set the CSS variable in `:host` to the value of `$searchbar-ios-input-border-radius`:

```css
:host {
  --border-radius: #{$searchbar-ios-input-border-radius};
}
```

Then, in `searchbar.scss` add the border radius to `.searchbar-input` there:

```css
.searchbar-input {
  @include border-radius(var(--border-radius));
}
```

This lets us set the iOS and MD border radius values while only defining the actual `border-radius` property once.